docs/openapi.json

```diff
@@ -156,6 +156,55 @@
         "summary": "TEMPORARY displays a message about how to access a resource."
       }
     },
+    "/api/logout": {},
+    "/api/me": {
+      "get": {
+        "description": "This resource provides basic information from an authenticated user based on the session cookie presence.",
+        "operationId": "get_me",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "User information correspoding to the session cookie sent in the request.",
+            "examples": {
+              "application/json": {
+                "email": "user@reana.info",
+                "reana_token": "Drmhze6EPcv0fN_81Bj-nA"
+              }
+            },
+            "schema": {
+              "properties": {
+                "email": {
+                  "type": "string"
+                },
+                "reana_token": {
```

Comment:
What do you think about only serving the email? In the UI, the SPA will have a cookie already (if logged in), and if someone uses the REST API, they should be using the access token already so no need to return it.

Reply:
Discussed offline:
The REANA token can be useful there for users that log into the UI and grab the token in order to use `reana-client` for instance.